packages/sign_in_with_apple/README.md

```diff
@@ -38,6 +38,87 @@ SignInWithAppleButton(
 
 ## Integration
 
+Integrating Sign in with Apple goes beyond just adding this plugin to your `pubspec.yaml` and using the credential-receiving functions exposed by it.
+
+Once you receive the credentials, they need to the verified with Apple's servers (to ensure that they are valid and really concern the mentioned user) and then a new session should be derived from them in your system.
+
+Your server should then daily verify the session with Apple (via a refresh token it obtained on the initial validation), and revoke the session in your system if the authorization has been withdrawn on Apple's side.
+
+### Prerequisites
+
+Before you can start integrating (or even testing) Sign in with Apple you need a [paid membership to the Apple Developer Program](https://developer.apple.com/programs/). Sign in with Apple is one of the restricted services which is not available for free with just an Apple ID ([source](https://developer.apple.com/programs/whats-included/)).
+
+### Setup
+
+#### Register an App ID  
+
+If you don't have one yet, create a new one at https://developer.apple.com/account/resources/identifiers/list/bundleId following these steps:
+
+* Click "Register an App ID"
+* In the wizard select "App IDs", click "Continue"
+* Set the `Description` and `Bundle ID`, and select the `Sign In with Apple` capability
+  * Usually the default setting of "Enable as a primary App ID" should suffice here. If you ship multiple apps that should all share the same Apple ID credentials for your users, please consult the Apple documentation on how to best set these up.
+* Click "Continue", and then click "Register" to finish the creation of the App ID
+
+In case you already have an existing App ID that you want to use with Sign in with Apple:
+
+* Open that App ID from the list
+* Check the "Sign in with Apple" capability
+* Click "Save"
+
+If you have change your app's capabilities, you need to fetch the updated provisioning profiles (for example via Xcode) to use the new capabilities.
```

Comment:
Don't you need to recreate the provisioning profiles? Or is re-fetching really enough?

Reply:
I don't know for sure, as I haven't downloaded these yet (as I really did everything from scratch with a blank new account).

I will follow up with iOS and then macOS separately.

Reply:
Just FYI: While Apple doesn't mention it explicitly, the provisioning profiles are bound to the capabilities of an App while creating that profile. So you will need to recreate these profiles.

In case the user uses the `Automatically manage Signing` within XCode, recreating the profile is not needed as XCode will do it for him.